aten/src/ATen/native/cuda/foreach/Utils.cuh

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#define ILP 4
+#define CHUNK_SIZE 65536
+#define BLOCK_SIZE 1024
```

Comment:
These probably don't need to be macros, they can just be constexpr ints and will be just fine. Defining generic names for macros in a header is a double-no-no, since that greatly increases the chance you'll end up with a conflict with another header.